packages/xo-server/src/api/metadata-backup.mjs

```diff
@@ -135,6 +135,9 @@ restore.params = {
   id: {
     type: 'string',
   },
+  poolUuid: {
```

Comment:
XO API methods accepts object identifiers which might be different from the UUID (currently either UUID or opaque ref but it depends on the object's type and might change in the future).

You need to handle this by using something like `this.getXapiObject(id).uuid` and reflect this in the param name:

```suggestion
  poolId: {
```